project/Dependencies.scala

```diff
@@ -16,7 +16,7 @@ object Dependencies {
 
     val cats = "2.6.1"
 
-    val catsEffect = "2.5.1"
+    val catsEffect = "3.1.1"
```

Comment:
No, that's wont work. You can wait for #720 where both ce dependencies will be in the build

Reply:
Quote from the description:
> I expect some other PR to do version update the right way